newIDE/app/src/ProjectsStorage/LocalProjectWriter.js

```diff
@@ -80,6 +134,38 @@ export default class LocalProjectWriter {
     }
   };
 
+  static saveProjectAs = (project: gdProject): Promise<void> => {
+    const filepath = project.getProjectFile();
+    const filenameWithExtension = path.basename(project.getProjectFile()) || "game.json";
```

Comment:
`|| "game.json"` looks like a good idea - will be useful in case projects don't have a saved project file at some point :+1: